docusaurus.config.js

```diff
@@ -103,6 +104,10 @@ const config = {
                 label: 'Twitter',
                 href: TWITTER_URL,
               },
+              {
+                label: 'LinkedIn',
+                href: LINKEDIN_URL,
+              },
```

Comment:
@ArslanYM, You have these changes in PR #38, can you make sure PR has changes related to the mentioned issue?

Reply:
i dont understand what youre asking me to do?

Reply:
```suggestion
              {
                label: 'LinkedIn',
                href: LINKEDIN_URL,
              },
```

You have already made these changes in PR #38. so why we are again include these changes in this PR?